<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>972</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="grb_1">
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QGroupBox" name="gpb_2">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="11" column="2">
          <widget class="QComboBox" name="cmb_capture_method"/>
         </item>
         <item row="1" column="2">
          <widget class="QgsMapLayerComboBox" name="ml_outlines_layer"/>
         </item>
         <item row="6" column="2">
          <widget class="QLineEdit" name="le_data_description"/>
         </item>
         <item row="12" column="2">
          <widget class="QComboBox" name="cmb_organisation"/>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="lb_capture_method">
           <property name="text">
            <string>Capture Method:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QgsFieldComboBox" name="fcb_external_id"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lb_capture_src_grp">
           <property name="text">
            <string>Capture Source Group:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lb_external_id">
           <property name="text">
            <string>External ID Field:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QRadioButton" name="rad_external_source">
           <property name="text">
            <string>External Source Exists</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="lb_data_desc">
           <property name="text">
            <string>Data Description:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QComboBox" name="cmb_capture_src_grp"/>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="lb_organisation">
           <property name="text">
            <string>Organisation:</string>
           </property>
          </widget>
         </item>
         <item row="16" column="2">
          <layout class="QHBoxLayout" name="hzly_1">
           <item>
            <widget class="QPushButton" name="btn_bl_ok">
             <property name="text">
              <string>Ok</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_bl_reset">
             <property name="text">
              <string>Reset</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lb_outlines">
           <property name="text">
            <string>Bulk Outlines Layer:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="lb_external_id_2">
           <property name="text">
            <string>External ID:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QComboBox" name="cmb_external_id"/>
         </item>
         <item row="0" column="0" colspan="3">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>                                  Bulk Load Outline</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>30</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btn_compare_outlines">
        <property name="text">
         <string>Compare Outlines</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpb_edits">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16760000</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="rad_add">
          <property name="text">
           <string>Add Outline</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rad_edit">
          <property name="text">
           <string>Edit Outline</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lb_bulk_load_status">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>Bulk Load Status:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cmb_status"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="l_capture_method">
        <property name="text">
         <string>Capture Method:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cmb_capture_method_2"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="l_capture_source">
        <property name="text">
         <string>Capture Source:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="cmb_capture_source"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lb_town">
        <property name="text">
         <string>Town City:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="cmb_town"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lb_suburb">
        <property name="text">
         <string>Suburb Locality:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="cmb_suburb"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lb_ta">
        <property name="text">
         <string>Territorial Authority: </string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="cmb_ta"/>
      </item>
      <item row="7" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="btn_edit_ok">
          <property name="text">
           <string>Ok</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_edit_reset">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_edit_cancel">
          <property name="text">
           <string>Exit Editing</string>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
     <zorder>cmb_status</zorder>
     <zorder>lb_bulk_load_status</zorder>
     <zorder>l_capture_method</zorder>
     <zorder>cmb_capture_method_2</zorder>
     <zorder>cmb_capture_source</zorder>
     <zorder>l_capture_source</zorder>
     <zorder>lb_town</zorder>
     <zorder>cmb_town</zorder>
     <zorder>lb_suburb</zorder>
     <zorder>cmb_suburb</zorder>
     <zorder>lb_ta</zorder>
     <zorder>cmb_ta</zorder>
     <zorder></zorder>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_alter_rel">
     <property name="text">
      <string>Alter Relationships</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_publish">
     <property name="text">
      <string>Publish</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_exit">
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="vtsp_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <spacer name="vtsp_1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsfieldcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
